src/Output.zig

```diff
@@ -31,10 +34,14 @@ pub fn create_surface(output: *Output) !void {
     const lock_surface = try output.lock.session_lock.?.getLockSurface(surface, output.wl_output);
     lock_surface.setListener(*Output, lock_surface_listener, output);
     output.lock_surface = lock_surface;
+
+    const viewport = try output.lock.viewporter.?.getViewport(surface);
+    output.viewport = viewport;
```

Comment:
unlike the lines above, line length isn't an issue here.
```suggestion
    output.viewport = try output.lock.viewporter.?.getViewport(surface);
```